docs/howto/garbage-collection/index.md

```diff
@@ -1,32 +1,32 @@
 ---
 title: Garbage Collection
 description: Clean up expired objects using the garbage collection feature in lakeFS.
-parent: Garbage Collection
-grand_parent: How-To
-nav_order: 1
-has_children: false
+parent: How-To
+has_children: true
 redirect_from:
   - /reference/garbage-collection.html
   - /howto/garbage-collection-index.html
```

Comment:
Should be removed right?

Reply:
what should be removed?

Reply:
The line - you deleted the file docs/howto/garbage-collection-index.md, shouldn't we remove this redirect as well?

Reply:
the redirect is necessary for any external links into /howto/garbage-collection-index.html otherwise they'll get a 404.